tests/sentry/integrations/github/test_search.py

```diff
@@ -13,7 +13,7 @@ class GithubSearchTest(APITestCase):
     provider = "github"
     base_url = "https://api.github.com"
 
-    def create_integration(self):
+    def create_slack_integration(self):
```

Comment:
I think this was an accident when you renamed the fixture

Reply:
oops